pyinstrument/renderers/console.py

```diff
@@ -22,22 +24,25 @@ def __init__(
         self,
         unicode: bool = False,
         color: bool = False,
+        flat: bool = False,
         time: LiteralStr["seconds", "percent_of_total"] = "seconds",
         **kwargs: Any,
-    ):
+    ) -> None:
         """
         :param unicode: Use unicode, like box-drawing characters in the output.
         :param color: Enable color support, using ANSI color sequences.
         :param time: How to display the duration of each frame - ``'seconds'`` or ``'percent_of_total'``
+        :param flat: Display a flat profile instead of a call graph.
```

Comment:
I'd suggest keeping them in the same order as in the `__init__` parameters.

```suggestion
        :param flat: Display a flat profile instead of a call graph.
        :param time: How to display the duration of each frame - ``'seconds'`` or ``'percent_of_total'``
```